demo/src/app/components/rating/demos/basic/basic.component.html

```diff
@@ -0,0 +1,7 @@
+<ngb-rating #ngbRating [(rate)]="rate" [max]="max" [readonly]="isReadonly" (hover)="hoveringOver($event)" aria-labelledby="default-rating"></ngb-rating>
+<span class="label {{percent < 30 && 'label-warning'}} {{percent >= 30 && percent < 70 && 'label-info'}} {{percent >= 70 && 'label-success'}}" *ngIf="overStar && !isReadonly">{{percent}}%</span>
```

Comment:
Do we want to have those color changing labels in the simple demo? I feel like it is cluttering things without providing too much added value... Thoughts?